packages/vm/lib/evm/evm.ts

```diff
@@ -92,6 +92,14 @@ export function COOGResult(gasUsedCreateCode: BN): ExecResult {
   }
 }
 
+export function EIP3541ViolationResult(gasLimit: BN): ExecResult {
```

Comment:
Maybe I am missing something, but where is this function actually called? 😜

Reply:
Heh. Oops.